sway/sway-input.5.scd

```diff
@@ -88,7 +88,7 @@ The following commands may only be used in the configuration file.
 	Maps inputs from this device to the specified output. Only meaningful if the
 	device is a pointer, touch, or drawing tablet device.
 
-*input* <identifier> map_to_region <WxH@X,Y>
+*input* <identifier> map_to_region <X> <Y> <width> <height>
```

Comment:
It seems like you can't have commas in command invocations, so I had to change this. It's similar to `grim`'s output format, but with spaces as all separators.